CMSIS/Core/Include/cmsis_armcc.h

```diff
@@ -2,7 +2,7 @@
  * @file     cmsis_armcc.h
  * @brief    CMSIS compiler ARMCC (Arm Compiler 5) header file
  * @version  V5.1.1
- * @date     30. July 2019
+ * @date     28. January 2020
```

Comment:
I'd like to ask you to increase the minor version of all touched files as well, e.g. `5.2.0` in this case.